inst/qml/RobustBayesianMetaAnalysis.qml

```diff
@@ -65,8 +65,8 @@ Form
 		RadioButton
 		{
 			label:		qsTr("Cohen's d / t-statistics & N / SE")
-			value:		"cohensd"
-			id:			measures_cohensd
+			value:		"cohensD"
+			id:			measuresCohensD
```

Comment:
perhaps this is just github, but there seems to be an extra tab/white space here